shell/utils/create-yaml.js

```diff
@@ -43,14 +43,33 @@ export const NEVER_ADD = [
   'metadata.initializers',
   'metadata.managedFields',
   'metadata.ownerReferences',
-  'metadata.resourceVersion',
+  // 'metadata.resourceVersion',
   'metadata.relationships',
   'metadata.selfLink',
+  'metadata.state',
   'metadata.uid',
+  'metadata.fields',
   // CRD -> Schema describes the schema used for validation, pruning, and defaulting of this version of the custom resource. If we allow processing we fall into inf loop on openAPIV3Schema.allOf which contains a cyclical ref of allOf props.
   'spec.versions.schema',
+  'spec.template.metadata.creationTimestamp',
   'status',
   'stringData',
+  'links',
+  '__clone',
+  '__active',
+  '__init',
+  '_init',
+  '_type',
+  'type',
+  'error',
+];
+
+export const NEVER_ADD_CONTAINER_FIELDS = [
```

Comment:
I opt for a constant. Services should not contain configurations.

Reply:
Given the fact that we are decoupling logic, I would rather move also the other configuration. This has not previously done to avoid layer of concerns in the PR and have a separated issue.
I would rather consider keeping the 2 configurations together, either separated configuration or keep them where they are.